pkg/ingester/ingester.go

```diff
@@ -620,6 +622,23 @@ func (i *Ingester) Push(ctx context.Context, req *logproto.PushRequest) (*logpro
 	return &logproto.PushResponse{}, err
 }
 
+// GetStreamRates returns a response containing all streams and their current rate
+func (i *Ingester) GetStreamRates(ctx context.Context, req *logproto.StreamRatesRequest) (*logproto.StreamRatesResponse, error) {
```

Comment:
WDYT of adding a TODO for us to, in the future, add a `sorted=true/false` flag to this call? Viewing the results sorted by streams that have the highest ratings have the potential to help a lot with debugging. But since it isn't the focus of this PR, I vote for it to just be a TODO.

Reply:
I don't know how useful this is for humans because we're only returning hashes rather than labels but I'm happy to add a TODO to remind us

Reply:
oh I forgot about the hash thing :facepalm: resolving this